### PR TITLE
Updatev2name

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [x] Ready

## Description
Changed in v2 integrations the display name to v2

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

File: Integrations/MISP_V2/CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 ## [Unreleased]
-
+-
 
 ## [20.2.4] - 2020-02-25
 Fixed default value for the ***PREDEFINED*** argument in the ***misp-search*** command.
+Changed display name to lower case v2
```
Comment (bakatzir) [inline Integrations/MISP_V2/CHANGELOG.md]:
```suggestion
```

**Comment by orhovy** [comment]
@anara123 can you please force merge this , the only changes in this branch is changing display name for some v2 integration and ci fails beacuse of autofocus test playbook which is irrelevent to this change.

**Comment by anara123** [comment]
@orhovy 
image

I think the new content wasn't uploaded properly to the instance.
Please make sure that the content has been uploaded

**Comment by orhovy** [comment]
@anara123 checked it , now the new content was uploaded properly and still test playbooks fail.

**Pull request merged**